recipes/defopt/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "defopt" %}
+{% set version = "6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/defopt-{{ version }}.tar.gz
+  sha256: a8fcd5560c845dceafd495d7dc97da355b1fdaa1881b5b83cf0fa4e9c7534285
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools_scm
+  run:
+    - docutils >=0.10
+    - python >=3.5
+    - sphinxcontrib-napoleon >=0.7.0
+    - colorama
+
+test:
+  imports:
+    - _defopt_version
```

Comment:
Is this part of the official API?

Reply:
I used `grayskull` as [recommended in the README](https://github.com/conda-forge/staged-recipes#grayskull---recipe-generator-for-python-packages-on-pypi).  I believe it's pulling this import from https://github.com/anntzer/defopt/blob/66731cf25496f515303587744fc9ce604463acde/setup.py#L16.  We can leave it out, as I dont' think it's strictly necessary.  Thoughts?

Reply:
please remove

Reply:
Done